crates/client/analytics/src/lib.rs

```diff
@@ -196,6 +196,24 @@ pub fn register_counter_metric_instrument<T: CounterType<T> + Display>(
     T::register_counter(crate_meter, instrument_name, desc, unit)
 }
 
+pub trait UpDownCounterType<T> {
```

Comment:
would be great to document those functions

Reply:
NA